### ISSUE TITLE
Does lb support NS_MOREFRAG?

### ISSUE THREAD

**Issue by emillynge** [luigirizzo/netmap#674]
Hi Maintainers.

We are having some issues relating to `lb` on machines with MTU > 1500.
What we are seeing is very similar to https://github.com/luigirizzo/netmap/issues/371 where the same log line is just repeated. (I believe this has to do with the main loop crashing leaving the logging thread to log the same data over and over)

My suspicion is that `lb` somehow breaks when it encounter a packet with a length above the `buf_size`. Looking at the code I also notice no usage of the `NS_MOREFRAG` flag anywhere.

1) Am I correct in concluding that `lb` does *not* know how to handle packets which is broken up into fragments?

2) If I want to implement handling of `NS_MOREFRAG`, where would that need to happen? (I am guessing the `forward_packet` needs changes as well as the loop that is calling the `pkt_hdr_hash` and  `forward_packet`)

Regards

**Comment by giuseppelettieri** [comment]
You are right, ```lb``` doesn't handle ```NS_MOREFRAG```. They quickest way to overcome the problem, if you have enough memory, is to increase the size of the netmap buffers so that jumbo frames are not fragmented (just write into ```/sys/module/netmap/parameters/buf_size``` and restart your netmap apps).

Of course, ```NS_MOREFRAG``` support is welcome and you have correctly identified the places that need some work. The hash must be computed only on the first fragment and you must be careful not to forward a packet until you have all the fragments. Please note that netmap should guarantee (modulo bugs) that each RXSYNC only gives you complete packets.

**Comment by giuseppelettieri** [comment]
I have finally found the time to add support for ```NS_MOREFRAG``` in ```lb```, in case you are still interested. It's in the current master.

**Comment by emillynge** [comment]
That's awesome!

Thank you very much. It is absolutely something I'm interested in, though we may cherry pick the feature back to APIv13 since we are have some older netmap applications.

BTW, we have found that we needed to strip away ERSPAN headers before sending to the first pipe group. 
Would that type of feature be welcome in lb as a PR?

**Comment by giuseppelettieri** [comment]
If it's not too much trouble I would gladly take a look at the code. I wonder if adding support for offsets would help in any way. A PR will do just fine, thanks!
